sql/updates/pre-0.7.0--0.8.0-dev.sql

```diff
@@ -1,3 +1,24 @@
 DROP FUNCTION _timescaledb_internal.create_hypertable_row(REGCLASS, NAME, NAME, NAME, NAME, INTEGER, NAME, NAME, BIGINT, NAME, REGPROC);
 DROP FUNCTION _timescaledb_internal.rename_hypertable(NAME, NAME, NAME, NAME);
 DROP FUNCTION _timescaledb_cache.invalidate_relcache(oid);
+
+WITH ind AS (
```

Comment:
Why do we need this? Didn't upgrades break if they had the foreign key thing? So, what is there to fix up?